### ISSUE TITLE
TrailingComma still registers an offense when it shouldn't

### ISSUE THREAD

**Issue by jfelchner** [rubocop/rubocop#1167]
I noticed a bug from the fix that @jonas054 did in #1075.  Unfortunately this is still registering an offense:

``` ruby
scout.throttle!(status:     'realtime',
                over_limit: true)
```

Even when I have the config set like so:

``` yaml
TrailingComma:
  EnforcedStyleForMultiline: 'comma'
```

I may be wrong, but it seems like params to a method call may need to be handled differently to just a regular hash since:

``` ruby
scout.throttle! status:     'realtime',
                over_limit: true
```

_Cannot_ have a trailing `,`, even though there's no `)` on that last line, otherwise it results in a parse error.

Here's the failing spec to help you out:

``` ruby
    context 'when EnforcedStyleForMultiline is comma' do
      it 'accepts a method call with Hash as last parameter split on multiple lines' do
        inspect_source(cop, ['some_method(a: "b",',
                             '            c: "d")'])
        expect(cop.offenses).to be_empty
      end
    end
```
